### PR TITLE
Test/utilities

### PR DESCRIPTION
**Pull request by m4rtint**

### CODE REVIEW

**Comment by mikeslgoh** [comment]
So why the decision to not keep it as a static class?

**Comment by m4rtint** [comment]
@mikeslgoh you can't mock static classes

**Comment by m4rtint** [comment]
Oh wait.. Hmm I see what you mean.
Yeah I don't havr an answer for that.

**Comment by m4rtint** [comment]
I think. It's just better for testing.
Cuz with static class, there's no absolute clean way to null a static class.. Unless you create a custom class to clean out the state of the static class.

File: Assets/Tests/Utilities_Tests.cs [hunk 1]
```diff
@@ -0,0 +1,258 @@
...
+    public class Utilities_Tests
+    {
+        private AngryGooseGames.Utilities utility;
+        [SetUp]
+        public void Setup()
+        {
+            utility = new AngryGooseGames.Utilities();
+        }
+
+        [TearDown]
```
Comment (Gursaanj) [inline Assets/Tests/Utilities_Tests.cs]: what Does [TearDown} do?

Comment (m4rtint) [reply Assets/Tests/Utilities_Tests.cs]: It's a method that is ran every single time a test has been completed. 
Lmk if u want more details on why or how to use it.

**Comment by Gursaanj** [comment]
[Refactor Utility Class for testing](https://trello.com/c/jw2Q2YrJ/310-refactor-utility-class-for-testing)

**Pull request merged**